### PR TITLE
python39Packages.cryptography: switch to pytestCheckHook, little cleanup, add SuperSandro2000 as maintainer

### PR DESCRIPTION
**Pull request by SuperSandro2000**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Review by primeos — APPROVED** [review]
LGTM based on a quick look and assuming that the builds still pass. Thanks for improving the packaging :)

Would you mind actively maintaining this package? Then I'd remove myself as maintainer (I've been planning to do that for a while now but it's an important and security-critical package that shouldn't be unmaintained).

**Comment by SuperSandro2000** [comment]
> Would you mind actively maintaining this package? Then I'd remove myself as maintainer (I've been planning to do that for a while now but it's an important and security-critical package that shouldn't be unmaintained).

Na, no problem. Do you have anything else you want to hand over?

File: pkgs/development/python-modules/cryptography/vectors.nix
Comment (SuperSandro2000) [inline pkgs/development/python-modules/cryptography/vectors.nix:21]:
```suggestion
    maintainers = with maintainers; [ SuperSandro2000 ];
```

File: pkgs/development/python-modules/cryptography/default.nix
Comment (SuperSandro2000) [inline pkgs/development/python-modules/cryptography/default.nix:90]:
```suggestion
    maintainers = with maintainers; [ SuperSandro2000 ];
```

**Comment by primeos** [comment]
> Na, no problem.

Cool, thanks :)

> Do you have anything else you want to hand over?

Hehe, plenty... :o :D
Luckily I've already managed to get rid of most of the packages that I don't use anymore :)

**Pull request merged**
